client/components/auth/auth.service.js

```diff
@@ -152,6 +156,7 @@ function AuthService($location, $http, $cookieStore, $q, appConfig, Util, User)
      */
     getCurrentUser(callback) {
       if (arguments.length === 0) {
+        smallgroup._id ? currentUser.smallgroup = true : currentUser.smallgroup = false;
```

Comment:
I think a clearer way to write this might be `currentUser.smallgroup = smallgroup._id !== null;`